Firestore/Source/Remote/FSTRemoteEvent.mm

```diff
@@ -248,33 +260,33 @@ @interface FSTRemoteEvent () {
 }
 
 - (instancetype)
-initWithSnapshotVersion:(FSTSnapshotVersion *)snapshotVersion
+initWithSnapshotVersion:(SnapshotVersion)snapshotVersion
           targetChanges:(NSMutableDictionary<FSTBoxedTargetID *, FSTTargetChange *> *)targetChanges
         documentUpdates:(std::map<DocumentKey, FSTMaybeDocument *>)documentUpdates;
 
-@property(nonatomic, strong) FSTSnapshotVersion *snapshotVersion;
-
 @end
 
 @implementation FSTRemoteEvent {
   std::map<DocumentKey, FSTMaybeDocument *> _documentUpdates;
+  SnapshotVersion _snapshotVersion;
 }
+
 + (instancetype)
-eventWithSnapshotVersion:(FSTSnapshotVersion *)snapshotVersion
+eventWithSnapshotVersion:(SnapshotVersion)snapshotVersion
```

Comment:
If you ever see these kinds of static factories in internal code, just delete them.

See background here: https://github.com/firebase/firebase-ios-sdk/pull/1167#pullrequestreview-114916597

Reply:
done